integration/resources/templates/combination/connector_appsync_to_table.yaml

```diff
@@ -103,23 +105,47 @@ Resources:
             `,
           };
 
+          const fetch = async (url, options) =>
+            new Promise((resolve, reject) => {
+              const req = https.request(url, options, (res) => {
```

Comment:
😭

Reply:
yes, fetch available only v17 on and it's awful to get around without 3rd party packages